tests/fixtures/test_invalid.yaml

```
@@ -0,0 +1,90 @@
+        buckets:
+          - some_bucket  # This should be an ERROR - final steps shouldn't have buckets
+        transitions:
+          some_bucket:
+            content_blocks:
+              - "Done"
+            # No next_section_and_step - this makes it a terminal step
+      
+      - step_4  # Missing step_id field - ERROR
+        title: "Invalid Step Structure"
+        # Missing either content_blocks or question - ERROR
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)